generators/app/index.js

```diff
@@ -134,12 +167,39 @@ module.exports = class extends BaseGenerator {
         });
     }
 
+    getPreviousKafkaConfiguration() {
+        try {
+            const previousGlobalConfiguration = jsYaml.safeLoad(
+                fsModule.readFileSync(`${jhipsterConstants.SERVER_MAIN_RES_DIR}config/application.yml`, 'utf8')
+            );
+            if (previousGlobalConfiguration && previousGlobalConfiguration.kafka) {
+                return previousGlobalConfiguration.kafka;
+            }
+        } catch (e) {
+            this.log(
+                `${chalk.red.bold(
+                    'WARN!'
+                )} Could not parse the previous kafka configuration, some previous configurations could be overwritten\n`
```

Comment:
`Kafka` instead of `kafka` and `the previous configuration` instead `some previous configurations`?

Reply:
Yes i gonna change it